data/tools/ale.yml

```diff
@@ -0,0 +1,10 @@
+name: ale
+category:
+  - linter
+tags: []
+license: BSD 2-Clause "Simplified" License
+type:
```

Comment:
Seeing this example, I think it should be `types` if it's a list. `type` would indicate to me that it's a single value.